datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -316,7 +323,6 @@ CREATE TABLE dcsa_im_v3_0.equipment (
     equipment_reference varchar(15) PRIMARY KEY,    -- The unique identifier for the equipment, which should follow the BIC ISO Container Identification Number where possible. According to ISO 6346, a container identification code consists of a 4-letter prefix and a 7-digit number (composed of a 3-letter owner code, a category identifier, a serial number and a check-digit). If a container does not comply with ISO 6346, it is suggested to follow Recommendation #2 “Container with non-ISO identification” from SMDG.
     -- Unique code for the different equipment size/type used for transporting commodities. The code is a concatenation of ISO Equipment Size Code and ISO Equipment Type Code A and follows the ISO 6346 standard.
     iso_equipment_code char(4) NULL REFERENCES dcsa_im_v3_0.iso_equipment_code (iso_equipment_code),
-    is_shipper_owned boolean NOT NULL DEFAULT false,
```

Comment:
This is used by EBL. If it is removed, then we need a replacement for EBL.

Reply:
@raisoman  had commented that it's not supposed to be in equipment, maybe he can clarify.
However note that `is_shipper_owned` its not part of `equipment` in IM 3.1.